### DISCUSSION TITLE
blockchain_v1_mainnet.sqlite is too big , space of disk C is not enough.

### DISCUSSION THREAD

**Discussion by mapleaj** [Chia-Network/chia-blockchain#7542] [Support]
Can we move C:\Users\Username\.chia to another disk? Most of people have not enough space in disk C.

**Comment by mapleaj** [comment]
blockchain_v1_mainnet.sqlite 7.66GiB
./chia is totle 10.8GiB

In the future it's may more than 1TiB,did you  image it? Now we need move this dir to another disk.It's growing too fast.

**Comment by MaximusChia** [comment]
That also cpncern me... how will be in the future? and if someone have to sync it from zero....

**Comment by d1Mm** [comment]
You can change data dir location by setting **CHIA_ROOT** environment variable or create a symbolic link
